src/community/label_propagation.c

```diff
@@ -103,11 +503,14 @@
  *   and igraph may renumber labels. However, co-membership constraints
  *   will be respected: two vertices can be fixed to be in the same or in
  *   different communities.
- * \param modularity If not a null pointer, then it must be a pointer
- *   to a real number. The modularity score of the detected community
- *   structure is stored here. Note that igraph will calculate the
- *   \em directed modularity if the input graph is directed, even if
- *   you set \p mode to \c IGRAPH_ALL
+ * \param lpa_variant Which variant of label propagation algorithm to run.
+ *   One of
+ *   \c IGRAPH_LPA_DOMINANCE Check for dominance of all nodes after each
+ *                           iteration.
+ *   \c IGRAPH_LPA_RETENTION Keep current label if among dominant labels,
+ *                           only check if labels changed.
+ *   \c IGRAPH_LPA_FAST      Sample from dominant labels, only check
+ *                           neighbors.
```

Comment:
Can you elaborate a bit more on what these mean in the main text? Make sure to define "dominant label" explicitly.

Reply:
Sure, I've added an explanation.